newsfragments/168.bugfix

```diff
@@ -0,0 +1 @@
+Fixed FormatNexus for Dectris Eiger using updated filewriter.
```

Comment:
This didn't affect before, right? Is it worth explicitly mentioning that this affects new Eiger firmwares?

Reply:
before they were on nearly nexus format => not an issue